frontend/src/lib/components/ActivationSidebar/ActivationLogic.ts

```diff
@@ -0,0 +1,238 @@
+import { kea, path, actions, selectors, connect, reducers, listeners, events } from 'kea'
+import { loaders } from 'kea-loaders'
+import { router, urlToAction } from 'kea-router'
+import api from 'lib/api'
+import { inviteLogic } from 'scenes/organization/Settings/inviteLogic'
+import { membersLogic } from 'scenes/organization/Settings/membersLogic'
+import { pluginsLogic } from 'scenes/plugins/pluginsLogic'
+import { teamLogic } from 'scenes/teamLogic'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { EventDefinitionType } from '~/types'
+
+import type { activationLogicType } from './activationLogicType'
+
+export enum ACTIVATION_TASKS {
```

Comment:
Nit: we usually don't capitalize enums, and should probably follow the standard here as well